microgateway_decorator/bin/compile

```diff
@@ -7,12 +7,95 @@ BUILD_DIR=$1
 CACHE_DIR=$2
 ENV_DIR=$3
 
+# make sure that the APIGEE_MICROGATEWAY_NODEJS_LOCAL_INSTALL variable is
+# true or false and if it is not set then assume that it is false.
+function check_nodejslocalinstall_env_var
+{
+  echo "checking APIGEE_MICROGATEWAY_NODEJS_LOCAL_INSTALL variable."
+  if [ "$APIGEE_MICROGATEWAY_NODEJS_LOCAL_INSTALL" != true ] && [ "$APIGEE_MICROGATEWAY_NODEJS_LOCAL_INSTALL" != false ]; then
```

Comment:
See comment below about local install config.

Reply:
Agreed will remove the APIGEE_MICROGATEWAY_NODEJS_LOCAL_INSTALL variable and rely on APIGEE_MICROGATEWAY_NODEJS_FILENAME instead.